copter/source/docs/failsafe-landing-page.rst

```diff
@@ -20,4 +20,7 @@ The main failsafe topics are listed below.
     Polygon Fence <polygon_fence>
     Crash Check <crash_check>
     Parachute <parachute>
-    ADSB Avoidance of manned aircraft <common-ads-b-receiver>
\ No newline at end of file
+    ADSB Avoidance of manned aircraft <common-ads-b-receiver>
+    Hardware Watchdog <common-watchdog>
```

Comment:
Is it perhaps a "CPU watchdog" instead of "Hardware Watchdog"?

Reply:
Independent?

Reply:
let's call it the "Independent Watchdog" and maybe let's include this video from Marco? https://www.youtube.com/watch?v=ZGuTIPLI_e0

Reply:
done...